lib/tsconfig.json

```diff
@@ -25,8 +25,7 @@
         "inlineSources": true
     },
     "files": [
-        "public_api.ts",
-        "node_modules/zone.js/dist/zone.js.d.ts"
+        "src/cashmere.ts"
```

Comment:
Does this file exist? It looks like public_api.ts is in this folder now.

Reply:
yup, I'll fix that